packages/@cosmos-building-blocks/app-ec2/.eslintrc

```diff
@@ -18,8 +18,10 @@
         "imports": "always-multiline",
```

Comment:
The think we need to move lint out to root, as their should only be one "standard" ?

Reply:
not sure I want the overrides to be global.  but good discussion to have